### PR TITLE
bundle-size: add tests for `app.js`

### LINKED ISSUE #3: Write tests for bundle-size app

### PR DESCRIPTION
**Pull request by danielrozenberg**
Fixes #3

I would appreciate a thorough review, since the rest of this app is going to rely on these tests being good :)

### CODE REVIEW

File: bundle-size/setup-db.js [hunk 1]
```diff
@@ -14,29 +14,43 @@
...
-  return db.destroy();
-});
+if (require.main === module) {
+  const db = dbConnect();
+  setupDb(db).then(() => {
+    log.info('Database table `checks` created.');
+  }).catch(error => {
+    log.error(error.message);
+  }).then(() => {
+    db.destroy();
```
Comment (erwinmombay) [inline bundle-size/setup-db.js]: dumb question on my part, does this destroy the actual db or just the connection?

File: bundle-size/test/app.test.js [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        repo: 'amphtml',
+        pull_request_id: 19621,
+        installation_id: 123456,
+        check_run_id: 555555,
+        delta: null,
+      },
+    ]);
+  });
+
+  test('approved review submitted by an OWNERS person', async () => {
```
Comment (erwinmombay) [inline bundle-size/test/app.test.js]: I'm a little unclear on what the resulting behavior should be by these actions

File: bundle-size/test/app.test.js [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+            },
+          });
+          return true;
+        })
+        .reply(200);
+
+    await probot.receive({name: 'pull_request_review', payload});
+  });
+
+  test('approved review submitted by a non-OWNERS person', async () => {
```
Comment (erwinmombay) [inline bundle-size/test/app.test.js]: a little unclear on what the resulting behavior should be on non-OWNER vs OWNER

File: bundle-size/setup-db.js (hunk 1 above)
Comment (rsimha) [reply bundle-size/setup-db.js]: From the line `exports.dbConnect = () => db;` above, seems like this is just a connection?

Comment (danielrozenberg) [reply bundle-size/setup-db.js]: This only destroys the connection

File: bundle-size/test/app.test.js (hunk 3 above)
Comment (danielrozenberg) [reply bundle-size/test/app.test.js]: I changed all the test names to be more descriptive

**Comment by danielrozenberg** [comment]
Good point re skipped message. I'll make a separate PR for this change

**Pull request merged**